.github/workflows/smalltalk-ci.yml

```diff
@@ -5,22 +5,20 @@ jobs:
   test:
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest, windows-latest]
+        os: [ ubuntu-latest, macos-latest, windows-latest ]
         smalltalk:
           - Squeak64-5.3
-          - Squeak64-5.2
-          - Squeak64-5.1
           - Squeak64-trunk
     runs-on: ${{ matrix.os }}
     name: ${{ matrix.smalltalk }} on ${{ matrix.os }}
     steps:
-      - uses: actions/checkout@v2
-      - uses: hpi-swa/setup-smalltalkCI@v1
-        id: smalltalkci
-        with:
-          smalltalk-image: ${{ matrix.smalltalk }}
-      - run: smalltalkci -s ${{ matrix.smalltalk }}
-        shell: bash
-        timeout-minutes: 15
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - uses: actions/checkout@v2
+    - uses: hpi-swa/setup-smalltalkCI@v1
+      id: smalltalkci
+      with:
+        smalltalk-version: ${{ matrix.smalltalk }}
+    - run: smalltalkci -s ${{ steps.smalltalkci.outputs.smalltalk-version }}
+      shell: bash
+      timeout-minutes: 15
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This is an old version (`smalltalk-image` is deprecated), I guess the changes were accidental or Squeak does weird stuff again?